src/main/java/net/atos/zac/app/informatieobjecten/converter/RESTInformatieobjectConverter.java

```diff
@@ -225,10 +231,10 @@ public EnkelvoudigInformatieobjectWithInhoudAndLock convert(
         enkelvoudigInformatieobjectWithInhoudAndLock.setLock(lock);
 
         if (restEnkelvoudigInformatieObjectVersieGegevens.status != null) {
-            enkelvoudigInformatieobjectWithInhoudAndLock.setStatus(InformatieobjectStatus.valueOf(restEnkelvoudigInformatieObjectVersieGegevens.status));
+            enkelvoudigInformatieobjectWithInhoudAndLock.setStatus(InformatieobjectStatus.fromValue(restEnkelvoudigInformatieObjectVersieGegevens.status));
         }
         if (restEnkelvoudigInformatieObjectVersieGegevens.vertrouwelijkheidaanduiding != null) {
-            enkelvoudigInformatieobjectWithInhoudAndLock.setVertrouwelijkheidaanduiding(Vertrouwelijkheidaanduiding.valueOf(
+            enkelvoudigInformatieobjectWithInhoudAndLock.setVertrouwelijkheidaanduiding(Vertrouwelijkheidaanduiding.fromValue(
```

Comment:
fromValue geeft een exception als het null is. Weet je zeker dat vertrouwelijkheidaanduiding nooit null is.